src/DataTypes/Serializations/ISerialization.cpp

```diff
@@ -172,6 +175,12 @@ String getNameForSubstreamPath(
             else
                 stream_name += "." + it->tuple_element_name;
         }
+        else if (it->type == Substream::VariantDiscriminators)
+            stream_name += ".discr";
```

Comment:
Yeah, let's change to `.variant_discr`. BTW, `variant_offsets` name will actually newer be used/displayed, because we don't serialize variant offsets. `VariantOffsets` stream was added only to be able to use substream cache for offsets during deserialization.